### DISCUSSION TITLE
What endpoint do I need to use to access customer transaction data?

### DISCUSSION THREAD

**Discussion by blipchin26** [etsy/open-api#1034] [Q&A] [answered]
I have received a valid access code and am trying to do the final step of this process in order to access transaction data. However, for whatever reason, my endpoint is wrong and I cannot for the life of me figure out the correct way to access the endpoint. I am printing out the following lines for bug testing:
print("Failed to fetch customer data:", response.status_code)
        print(response.content)
and receiving this in the console:
Failed to fetch customer data: 404
b'{"error": "Resource not found"}\n'

which I believe means I am not utilizing a valid endpoint. Attached is how I'm referencing the endpoint. If anyone knows the problem please let me know!
![API endpoint transactions](https://github.com/etsy/open-api/assets/113556181/260272b1-0480-4a1c-8a49-d093a3342d3c)

**Comment by turboturtleD** [comment]
What endpoint are you trying to hit?

**Reply by blipchin26** [reply]
The scope I'm using is transactions_r if that helps at all. According to Etsy's API documentation I thought this would be the endpoint for customer transaction data but maybe I messed up the headers or URL I am honestly not too sure.

**Reply by turboturtleD** [reply]
What is the name of the endpoint as it’s noted in the API resource?

**Reply by turboturtleD** [reply, marked as answer]
Like “getShopReceiptTransactionsByShop”

**Reply by blipchin26** [reply]
Yup that was it thank you. Just could not find that reference anywhere but I had it typed out in a comment earlier in my code. Woops. Appreciate the help.
